ituploader.js

```diff
@@ -0,0 +1,46 @@
+const {ethers, Contract} = require("ethers");
+const crypto = require('crypto');
+const {EthStorage} = require("./dist/index.cjs.js");
+const {NodeFile} = require("./dist/file.cjs.js");
+const fs = require('fs');
```

Comment:
This doesn't seem to be related to the SDK. Should we create a separate repo for testing?

Reply:
The the same question here, what is the motivation of adding this script to the SDK?